modules/service/src/main/scala/lucuma/odb/service/ObservationService.scala

```diff
@@ -153,29 +172,110 @@ object ObservationService {
           pq.stream(which.argument, chunkSize = 1024).compile.toList
         }
 
+      // Select the observation ids from among the provided collection of ids which have an observing mode that does
+      // not match the `newMode`.
```

Comment:
Need to delete this comment, it's incorrect.  `selectObservingModes` selects and groups the observing mode types of all the provided observations.